### PR TITLE
system/input: add input tools keyboard support

### PR DESCRIPTION
**Pull request by Gary-Hobson**
## Summary
add input tools keyboard support

## Testing
Enable CONFIG_INPUT_UINPUT and CONFIG_SYSTEM_INPUT ,
you can use "input keyup/keydown <keycode>" to enter virtual key values on the command line

### CODE REVIEW

File: system/input/input.c [hunk 1]
```diff
@@ -237,7 +243,7 @@ static int input_button(int argc, char **argv)
     }
 
   button = strtoul(argv[1], NULL, 0);
-  fd = open("/dev/ubutton", O_WRONLY);
+  fd     = open("/dev/ubutton", O_WRONLY);
```
Comment (xiaoxiang781216) [inline system/input/input.c:246]: revert the change

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]
